qubes/ext/admin.py

```diff
@@ -47,6 +55,21 @@ def __init__(self):
             self.policy_cache = utils.PolicyCache(lazy_load=True)
             self.policy_cache.initialize_watcher()
 
+    # pylint: disable=too-few-public-methods
```

Comment:
I think this doesn't need to be repeated - once per class is enough (and after you added new method, maybe it's not necessary anymore?)